packages/@uppy/transloadit/src/AssemblyOptions.test.js

```diff
@@ -2,11 +2,11 @@ import { describe, expect, it } from '@jest/globals'
 import AssemblyOptions from './AssemblyOptions.js'
 
 describe('Transloadit/AssemblyOptions', () => {
-  it('Validates response from getAssemblyOptions()', async () => {
+  it('Validates response from assemblyOptions()', async () => {
```

Comment:
Don't we want to keep this test until the option is actually removed?

Reply:
If this test was testing the package, then yes. But this is testing `AssemblyOptions`, it has no knowledge of what is deprecated and what isn't, that distinction lives in the package. It always receives `assemblyOptions`, so this is correct.